service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/RequestStatementExtractor.java

```diff
@@ -50,20 +47,20 @@ public static String extractRecipient(MessageContext messageContext, Reference r
         if (referenceId.getResourceType().equals(ResourceType.Practitioner.name())) {
             return messageContext.getInputBundleHolder()
                 .getResource(referenceId)
-                .map(resource -> (Practitioner) resource)
-                .map(value -> RECIPIENT_PRACTITIONER + extractHumanName(value.getNameFirstRep()))
+                .map(Practitioner.class::cast)
+                .map(value -> "Recipient Practitioner: { " + extractHumanName(value.getNameFirstRep()) + " }")
```

Comment:
Omit the {}'s. These are meant to indicate a substitution in the spec pseudocode.